src/badguy/goldbomb.cpp

```diff
@@ -170,7 +168,7 @@ GoldBomb::active_update(float dt_sec)
     return;
   }
 
-  if ((tstate == STATE_FLEEING || tstate == STATE_CORNERED) && on_ground() && might_fall(FLEEING_MAX_DROP_HEIGHT+1))
+  if ((tstate == STATE_FLEEING || tstate == STATE_CORNERED) && on_ground() && might_fall(600+1))
```

Comment:
What about having the non-hitbox relative ledge behaviour modes as private static constant variables to use in situations like these?

Reply:
Can the 600 be moved to a constant again?

Reply:
What do you mean by this?

Reply:
In `WalkingBadguy` class, on the top:

```
static const float s_fleeing_max_drop_height;
```

for example. Same for the other 2. Behaviour would still be determined by `set_ledge_behavior()`, but objects will also have access to these constants.